Source/WebInspectorUI/UserInterface/Views/SourceCodeTextEditor.js

```diff
@@ -743,7 +743,9 @@ WI.SourceCodeTextEditor = class SourceCodeTextEditor extends WI.TextEditor
         if (!script)
             return;
 
-        let endPosition = this.currentPositionToOriginalPosition(new WI.SourceCodePosition(lineNumber + 1, 0));
+        // If the current content is minified code, only show pause locations within 100 characters.
+        let limitLocations = this._isProbablyMinified && !this.formatterSourceMap;
+        let endPosition = this.currentPositionToOriginalPosition(new WI.SourceCodePosition(limitLocations ? lineNumber : (lineNumber + 1), limitLocations ? 100 : 0));
```

Comment:
another option would be to do something like this
```
let line = this.line(lineNumber);
let endPosition = this.currentPositionToOriginalPosition(new WI.SourceCodePosition(lineNumber, Math.min(line.length, 100));
```
but IMO this is worse cause it means that non-minified/pretty-printed code is penalized if there's ever a line more than 100 characters long

Reply:
If we were to do something like that I would suggest a higher limit than 100 and that we exclude leading whitespace on a line to cover as much as possible. But I think not limiting pretty-printed code is the right thing to do.